### ISSUE TITLE
Update dart.py user_bundle key to "options" from "tags"

### ISSUE THREAD

**Issue by OnodOfTheNorth** [its-dart/dart-tools#3]
currently the functions update_task and create_task in dart.py throw key errors when looking for the key "tags". It looks like its been updated to "options" based on what i could find in the returned JSON.

I got the JSON return by adding this to the create_task function 
```
    user_bundle = _get_full_user_bundle(session)
    # Create a new dictionary with only the headers
    headers_only_bundle = {key: {} for key in user_bundle.keys()}

    # Write the headers only bundle to a JSON file
    with open("user_bundle_headers_only.json", "w") as fout:
        json.dump(headers_only_bundle, fout, indent=2)
    fout.close()
    with open("user_bundle.json", "w") as fin:
        json.dump(user_bundle["options"], fin, indent=2)
```

I was able to fix the error by changing lines 571 and 712 in dart.py 

from:
```
 tags = user_bundle["tags"]
```
to:
``` 
tags = user_bundle["options"]
```

thanks for your great work! dart has been an amazing tool in my time management belt.

**Comment by zswaff** [comment]
Hey @OnodOfTheNorth, thanks a lot for digging in and submitting this clear issue. As you can tell, this lib is still a bit early!

We will get a solution like you described merged and deployed in the next couple of days

**Comment by zswaff** [comment]
@OnodOfTheNorth I just released [dart-tools 0.4.4](https://pypi.org/project/dart-tools/0.4.4/).

Thanks again for your help with this.

Please let us know if there's anything else we can do to improve--your feedback is invaluable and we will always take it!
